README.md

```diff
@@ -110,13 +117,12 @@ Thanks goes to these wonderful people ✨✨:
     <td align="center"><a href="https://megamind77-coder.github.io/new_portfolio/"><img src="https://avatars3.githubusercontent.com/u/42925218?v=4" width="100px;" alt=""/><br /><sub><b>Ishan Shanware</b></sub></a><br /><a href="https://github.com/Canvasbird/canvasboard/commits?author=MegaMind77-coder" title="Code">💻</a></td>
   </tr>
 </table>
-
+-->
```

Comment:
please remove the commented lines.

Reply:
This block is commented because this done the same work which I can do using simple 4 lines. So Can I remove this block if you don't want this block. ?

Reply:
> This block is commented because this done the same work which I can do using simple 4 lines. So Can I remove this block if you don't want this block. ?

Yes, after we merge this PR, we won't require this block. So simply remove it.